go get github.com/go-redis/redis is now giving me errors

Issue tracker is used for reporting bugs and discussing new features. Please use
[stackoverflow](https://stackoverflow.com) for supporting issues.

<!--- Provide a general summary of the issue in the Title above -->

## Expected Behavior

The redis package should install as i did not touch anything in this environment and it worked yesterday

## Current Behavior

The redis module does not compile and gives this error:

> [wssgo 12/15] RUN go get github.com/go-redis/redis:
#0 9.280 # github.com/go-redis/redis
#0 9.280 /go/src/github.com/go-redis/redis/commands.go:355:62: undefined: StringIntMapCmd
#0 9.280 /go/src/github.com/go-redis/redis/commands.go:3125:78: undefined: StringIntMapCmd
#0 9.280 /go/src/github.com/go-redis/redis/commands.go:3132:9: undefined: NewStringIntMapCmd
#0 9.280 /go/src/github.com/go-redis/redis/ring.go:568:18: c.shards undefined (type *Ring has no field or method shards)
------
failed to solve: executor failed running [/bin/sh -c go get github.com/go-redis/redis]: exit code: 2

 *  The terminal process "C:\WINDOWS\System32\cmd.exe /d /c docker compose -f "docker-compose-nginx.yml" up -d --build" terminated with exit code: 17. 
 *  Terminal will be reused by tasks, press any key to close it. 

## Possible Solution

<!--- Not obligatory, but suggest a fix/reason for the bug, -->

## Steps to Reproduce
run go get github.com/go-redis/redis in any command line. ive got the same error in docker and in WSL Ubuntu

## Context (Environment)

A docker  image was compiled from the official golang image.

## Detailed Description

No changes were made, all i did was build the image this morning and i was receiving these errors.

--- a/commands.go
+++ b/commands.go
@@ -352,7 +352,7 @@ type Cmdable interface {
 	PubSubNumSub(ctx context.Context, channels ...string) *MapStringIntCmd
 	PubSubNumPat(ctx context.Context) *IntCmd
 	PubSubShardChannels(ctx context.Context, pattern string) *StringSliceCmd
-	PubSubShardNumSub(ctx context.Context, channels ...string) *StringIntMapCmd
+	PubSubShardNumSub(ctx context.Context, channels ...string) *MapStringIntCmd
 
 	ClusterSlots(ctx context.Context) *ClusterSlotsCmd
 	ClusterNodes(ctx context.Context) *StringCmd
@@ -3122,14 +3122,14 @@ func (c cmdable) PubSubShardChannels(ctx context.Context, pattern string) *Strin
 	return cmd
 }
 
-func (c cmdable) PubSubShardNumSub(ctx context.Context, channels ...string) *StringIntMapCmd {
+func (c cmdable) PubSubShardNumSub(ctx context.Context, channels ...string) *MapStringIntCmd {
 	args := make([]interface{}, 2+len(channels))
 	args[0] = "pubsub"
 	args[1] = "shardnumsub"
 	for i, channel := range channels {
 		args[2+i] = channel
 	}
-	cmd := NewStringIntMapCmd(ctx, args...)
+	cmd := NewMapStringIntCmd(ctx, args...)
 	_ = c(ctx, cmd)
 	return cmd
 }

--- a/ring.go
+++ b/ring.go
@@ -565,7 +565,7 @@ func (c *Ring) SSubscribe(ctx context.Context, channels ...string) *PubSub {
 	if len(channels) == 0 {
 		panic("at least one channel is required")
 	}
-	shard, err := c.shards.GetByKey(channels[0])
+	shard, err := c.sharding.GetByKey(channels[0])
 	if err != nil {
 		// TODO: return PubSub with sticky error
 		panic(err)

